src/github.com/getlantern/enproxy/conn_test.go

```diff
@@ -91,26 +100,23 @@ func doTestTLS(buffered bool, t *testing.T) {
 
 	doRequests(tlsConn, t)
 
-	if bytesReceived != 555 {
-		t.Errorf("Bytes received of %d did not match expected %d", bytesReceived, 555)
-	}
-	if bytesSent != 4968 {
-		t.Errorf("Bytes sent of %d did not match expected %d", bytesSent, 4968)
-	}
+	assert.Equal(t, 550, bytesReceived, "Wrong number of bytes received")
+	assert.Equal(t, 1580, bytesSent, "Wrong number of bytes sent")
```

Comment:
This line is always failing for me. The number of bytes sent always equals 1576 instead of the expected 1580. 

--- FAIL: TestTLSStreaming (0.31 seconds)
        Location:       conn_test.go:104
	Error:		Not equal: 1580 (expected)
			        != 1576 (actual)
	Messages:	Wrong number of bytes sent

=== RUN TestTLSBuffered
--- FAIL: TestTLSBuffered (0.32 seconds)
        Location:       conn_test.go:104
	Error:		Not equal: 1580 (expected)
			        != 1576 (actual)
	Messages:	Wrong number of bytes sent